flixel/FlxGame.hx

```diff
@@ -223,16 +223,16 @@ class FlxGame extends Sprite
 	/**
 	 * Instantiate a new game object.
 	 * 
-	 * @param	GameSizeX		The width of your game in game pixels, not necessarily final display pixels (see Zoom).
-	 * @param	GameSizeY		The height of your game in game pixels, not necessarily final display pixels (see Zoom).
+	 * @param	GameSizeX		The width of your game in game pixels, not necessarily final display pixels (see Zoom). If it is less than 0, HaxeFlixel will auto detect the width according to the current window.
```

Comment:
These docs are not quite accurate, they only account for the `< 0` case, not `== 0`.